pkg/virt-controller/watch/application.go

```diff
@@ -443,7 +456,9 @@ func (app *VirtControllerApp) shouldChangeRateLimiter() {
 // Update virt-controller log verbosity on relevant config changes
 func (vca *VirtControllerApp) shouldChangeLogVerbosity() {
 	verbosity := vca.clusterConfig.GetVirtControllerVerbosity(vca.host)
-	log.Log.SetVerbosityLevel(int(verbosity))
+	if err := log.Log.SetVerbosityLevel(int(verbosity)); err != nil {
+		log.Log.Warningf("failed up update log verbosity to %d: %v", verbosity, err)
+	}
```

Comment:
Looks like there should be an `else` or we would contradict the warning.

Reply:
Good catch, done.